purism/librem/5r4/kernel/6.1.10.nix

```diff
@@ -0,0 +1,14 @@
+{ buildLinux, fetchFromGitLab, ... }@args:
+buildLinux (args // rec {
+  defconfig = "librem5_defconfig";
+  version = "6.1.10-librem5";
```

Comment:
```suggestion
  version = "6.1.10";
```

Reply:
Also, the modDirVersion has to be `6.1.10-librem5` anyway. Do you think it makes sense to set the kernel version to just 6.1.10? It has substantial modifications compared to upstream 6.1.10

Reply:
I would rather expose the kernel in the name rather in the version since we need to cleanly parse the kernel version properly in different places.